modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -40,19 +48,18 @@ Possible values include:
 |Yes
 
 |Basic authentication
-a|When Basic authentication is selected, specify the user name and password to protect the HTTP listener.
+a|When Basic authentication selected, specify the user name and password to protect the HTTP listener.
 
-You need to note the password specified during the creation of the endpoint. After the configuration is saved, you cannot retrieve or modify the password.
+After the configuration or creation of the endpoint is saved, you cannot retrieve or modify the password.
 
 The backend applications that send JSON or XML payloads to the HTTP receive endpoint must pass the credentials in the HTTP request header.
 |
 
 |API Key
-a|When API Key is selected, specify the HTTP header name and API key to protect the HTTP listener.
+a|The API Key for the HTTP listener.
 
-You need to note the API key specified during the creation of the endpoint. After the configuration is saved, you cannot retrieve or modify the API key.
+When the API Key is selected, specify the HTTP header name and API key to protect the HTTP listener. After the configuration or creation of the endpoint is saved, you cannot retrieve or modify the API key.
```

Comment:
passive: "is selected" by whom or what, and when?

Reply:
ditto re "configuration or creation"

Reply:
Done!